src/current-refinements/current-refinements.ts

```diff
@@ -1,17 +1,28 @@
 import { Component, Input, Inject, forwardRef } from '@angular/core';
 
-import { connectCurrentRefinedValues } from 'instantsearch.js/es/connectors';
+import { connectCurrentRefinements } from 'instantsearch.js/es/connectors';
 import { BaseWidget } from '../base-widget';
 import { NgAisInstantSearch } from '../instantsearch/instantsearch';
-import { noop, capitalize } from '../utils';
+import { noop } from '../utils';
+
+export type CurrentRefinementsItem = {
```

Comment:
If you rely on [InstantSearch.js 3.1.0](https://unpkg.com/instantsearch.js@3.1.0/), you can add the property `label`.